dbt/models/location/location.tax.sql

```diff
@@ -230,3 +230,4 @@ LEFT JOIN wide
             ELSE pcl.year
         END = wide.year
     )
+WHERE pcl.year >= (SELECT MIN(year) FROM wide)
```

Comment:
Previously we had been missing this filter, which meant that `location.tax` had a different universe of years than the rest of the inputs to `location.vw_pin10_location_fill` (specifically, it contained all parcel years, even for years before the first year of tax data). I don't think this should affect other queries that consume this particular table, since prior to this change, everything before the minimum year was totally empty anyway.